### ISSUE TITLE
Will we use an ORM?

### ISSUE THREAD

**Issue by larongbingo** [team-noob-noob/libman#14]
Are we going to use an ORM or just straight SQL statements?

**Comment by larongbingo** [comment]
@team-noob-noob/team-noob-noob

**Comment by joshybara** [comment]
Since were not that worrying about deadlines, sticking with SQL statements must be a better choice

**Comment by BernabePosadas** [comment]
never used ORM yet but if it already has some anti-SQL injection we can use them so we wont need to make a module for cleaning SQL statements

**Comment by larongbingo** [comment]
> never used ORM yet but if it already has some anti-SQL injection we can use them so we wont need to make a module for cleaning SQL statements

Yeah, that's one of the bonus feats of some ORM is that they add some protection to SQL injection.

**Comment by joshybara** [comment]
we can do both if you want

**Comment by larongbingo** [comment]
We can only do that if the ORM allows for custom queries like Sequelize.
